docs/content/installation/csi_ug_requirements.md

```diff
@@ -37,33 +37,60 @@ Perform these steps for each worker node in Kubernetes cluster to prepare your e
    ```
 
    This file can be used for both Fibre Channel and iSCSI configurations. To support iSCSI, uncomment the last two lines in the file.
 
-   **Important:** The 99-ibm-attach.yaml configuration file overrides any files that already exist on your system. Only use this file if the files mentioned are not already created. <br />If one or more have been created, edit this yaml file, as necessary.
+   **Important:** The `99-ibm-attach.yaml` configuration file overrides any files that already exist on your system. Only use this file if the files mentioned are not already created. <br />If one or more have been created, edit this YAML file, as necessary.
 
-   Apply the yaml file.
+   Apply the YAML file.
 
    `oc apply -f 99-ibm-attach.yaml`
+
+3. Configure storage system connectivity.
+
+    1.  Define the host of each Kubernetes node on the relevant storage systems with the valid WWPN (for Fibre Channel) or IQN (for iSCSI) of the node.
+
+    2.  For Fibre Channel, configure the relevant zoning from the storage to the host.
     
-3. If needed, enable support for volume snapshots (FlashCopy® function) on your Kubernetes cluster.
+4. (Optional) If planning on using volume snapshots (FlashCopy® function), enable support on your Kubernetes cluster.
 
    For more information and instructions, see the Kubernetes blog post, [Kubernetes 1.20: Kubernetes Volume Snapshot Moves to GA](https://kubernetes.io/blog/2020/12/10/kubernetes-1.20-volume-snapshot-moves-to-ga/).
 
    Install both the Snapshot CRDs and the Common Snapshot Controller once per cluster.
 
-   The instructions and relevant yaml files to enable volume snapshots can be found at: [https://github.com/kubernetes-csi/external-snapshotter#usage](https://github.com/kubernetes-csi/external-snapshotter#usage)
+   The instructions and relevant YAML files to enable volume snapshots can be found at: [https://github.com/kubernetes-csi/external-snapshotter#usage](https://github.com/kubernetes-csi/external-snapshotter#usage)
 
-4. Configure storage system connectivity.
+5. (Optional) If planning on using volume replication (remote copy function), enable support on your Kubernetes cluster.
+
+    Install the following replication CRDs once per cluster.
+
+    ```
+    curl -O https://raw.githubusercontent.com/csi-addons/volume-replication-operator/v0.1.0/config/crd/bases/replication.storage.openshift.io_volumereplicationclasses.yaml
+    kubectl apply -f ./replication.storage.openshift.io_volumereplicationclasses.yaml
+    
+    curl -O https://raw.githubusercontent.com/csi-addons/volume-replication-operator/v0.1.0/config/crd/bases/replication.storage.openshift.io_volumereplications.yaml
+    kubectl apply -f ./replication.storage.openshift.io_volumereplications.yaml
+    ```
+
+6. (Optional) To use CSI Topology, any nodes in the cluster must have the following labels that introduce topology awareness:
+      - `topology.kubernetes.io/region`
+      - `topology.kubernetes.io/zone`
```

Comment:
@ArbelNathan is `region`/`zone` specifically a must? because this is how the sentence sounds to me

Reply:
no, they are not a must and the prefix key will change [here ](https://github.com/IBM/ibm-block-csi-driver/pull/373) as well..

Reply:
@rivkap then I think only the common prefix is the mandatory part of the label

Reply:
so just write:
"(Optional) To use CSI Topoogy, any nodes in thecluster must have the `topology.kubernetes.io` label, to introduce topology awareness."
?

Reply:
not sure about `any` and `the`, but in general, yes

Reply:
DONE

Reply:
@ArbelNathan are `all` nodes required to have such label?
is it "the `topology.kubernetes.io` label" or more like "a `topology.kubernetes.io` label"?

Reply:
more like 'a'.. 
btw, it will be only a `topology.block.csi.ibm.com` prefix label and we won't support the `topology.kubernetes.io` (for now)

Reply:
@rivkap FYI

Reply:
DONE

Reply:
I still see `all` and `the`. what I mean is that:
`all nodes in the cluster must contain the `topology.block.csi.ibm.com` label` should be something like:
`at least one node in the cluster must contain a `topology.block.csi.ibm.com` label`

Reply:
@oriyarde  Thank you. Next time please specify (like you did here) so that we can save on some back and forth and misunderstandings.
I still think that it should be "contain **the** `topology.block.csi.ibm.com` label" if this is the only label that is being accepted, it's specific.
I am updating the beginning of the sentence though.
DONE

Reply:
`if this is the only label that is being accepted, it's specific.`
this isn't really a label by itself, it's just a prefix, so "a <prefix> label" would be any label with such prefix, e.g. `topology.block.csi.ibm.com/demo-region`

Reply:
updated to label prefix, per my conversation with @ArbelNathan 
DONE

Reply:
`...one node in the cluster must contain the label prefix of topology.block.csi.ibm.com`
what this says is true, but the phrasing sounds strange to me, because the way I see it, a node can't contain a prefix - it can have a label that has a prefix.

how about something like:
`...one node in the cluster must have a label with a topology.block.csi.ibm.com prefix`

Reply:
I'll update to "label-prefix" with a hyphen.
I am also updating contain to have but I don't think it is critical
DONE

Reply:
`label-prefix` sounds strange to me as well, but ok